### ISSUE TITLE
Add v4 commands to the docs

### ISSUE THREAD

**Issue by Filnor** [SOBotics/Natty#38]

**Comment by Bhargav-Rao** [comment]
v3 commands like `why` isn't documented as well.

**Comment by Filnor** [comment]
In that case, we get a list of all undocumented commands and add them to the docs and Nattys command list itself IMO.

**Comment by Bhargav-Rao** [comment]
Yeah, that'd be better. [Now that we have a reply handler service](https://github.com/SOBotics/Natty/blob/master/src/main/java/in/bhargavrao/stackoverflow/natty/services/ReplyHandlerService.java#L17), writing documentation for that is easy.
